cartographer_ros/cartographer_ros/node_main.cc

```diff
@@ -28,12 +28,16 @@ DEFINE_string(configuration_directory, "",
 DEFINE_string(configuration_basename, "",
               "Basename, i.e. not containing any directory prefix, of the "
               "configuration file.");
-DEFINE_string(map_filename, "", "If non-empty, filename of a map to load.");
+DEFINE_string(load_state_filename, "",
```

Comment:
@ojura, please make the follow-up change in cartographer_fetch/demo_freight_localization.launch